var a = 10;
var b = 20;

try {
  console.log(a + c);
} catch (err) {
  console.log(`ERROR IS: ${err}`);
}
